koku/api/settings/utils.py

```diff
@@ -20,13 +21,42 @@
 from api.report.constants import URL_ENCODED_SAFE
 from api.settings.settings import COST_TYPES
 from api.settings.settings import DEFAULT_USER_SETTINGS
+from api.utils import DateHelper
 from koku.settings import KOKU_DEFAULT_COST_TYPE
 from koku.settings import KOKU_DEFAULT_CURRENCY
+from masu.processor import is_customer_large
+from masu.processor.tasks import UPDATE_SUMMARY_TABLES_QUEUE
+from masu.processor.tasks import UPDATE_SUMMARY_TABLES_QUEUE_XL
+from masu.processor.tasks import UPDATE_SUMMARY_TABLES_TASK
 from reporting.user_settings.models import UserSettings
+from reporting_common.models import DelayedCeleryTasks
 
 """Utilities for Settings."""
 
 
+def delayed_summarize_current_month(schema_name: str, provider_uuids: list, provider_type: str):
```

Comment:
This func definition seems out of place here. Is it possible to put this next to the model and the pre-delete method?